### ISSUE TITLE
User image selection sequence is not remembered.

### ISSUE THREAD

**Issue by abhayrsingh87** [darsh2/MultipleImageSelect#34]
User image selection sequence is not remembered. If user picks 4,6 and 2 image from picker. 
Returned list contains 2,4 and 6 (chronological order). This will help in cases where single piece of info fragmented into different images. User can pic multiple information to make full piece of info.

**Comment by darsh2** [comment]
Yes, this will be a useful enhancement to add. However it has been a long time since I have worked on this library. Development will still be on hold for a month at the worst as I have other urgent matters to tend to. I would appreciate any pull requests for the same.
